src/plugins/home/public/application/components/use_case_card.tsx

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React from 'react';
+import { EuiIcon } from '@elastic/eui';
+import { i18n } from '@osd/i18n';
+
+import { ContentManagementPluginStart } from '../../../../content_management/public';
+
+interface Config {
+  title: string;
+  description: string;
+  icon: string;
+}
+
+export const WORKSPACE_USE_CASES = {
```

Comment:
I think we may not need to maintain this config twice as it is similar to the `WORKSPACE_USE_CASES` in src/plugins/workspace/common/constants.ts. But I may also be doubtful about whether we should set home plugin depending on workspace.